src/reconcile/rec_row.c

```diff
@@ -907,7 +907,7 @@ __wt_rec_row_leaf(
             kpack = &_kpack;
             __wt_cell_unpack(session, page, cell, kpack);
             if (btree->huffman_key == NULL && kpack->type == WT_CELL_KEY &&
-              tmpkey->size >= kpack->prefix) {
+              tmpkey->size >= kpack->prefix && tmpkey->size != 0) {
                 /*
                  * The previous clause checked for a prefix of zero, which means the temporary
```

Comment:
If size cannot be zero, why not remove the assert below?

Reply:
635ba65